torch/masked/_ops.py

```diff
@@ -1523,8 +1530,8 @@ def logaddexp(
     if dtype is None:
         dtype = input.dtype
     if input.layout == torch.strided and other.layout == torch.strided:
-        mask_input = _combine_input_and_mask(logsumexp, input, input_mask)
-        mask_other = _combine_input_and_mask(logsumexp, other, other_mask)
```

Comment:
Am I missing something here? Should `logaddexp` really refer to `logsumexp` or is this an autocomplete error?

Reply:
I do think this is intentional here, as `_combine_input_and_mask` is using the op in the first position to look up the identity element for reduction, there is no value to find for `logaddexp` so the use of `logsumexp` here looks deliberate.